config/initializers/active_admin.rb

```diff
@@ -230,6 +230,13 @@
   #     end
   #   end
 
+  config.namespace :admin do |admin|
```

Comment:
Couldn't see a link in admin on review app, but maybe was looking in the wrong place :shrug: 

Unrelated, Still keen to get away from Active admin eventually?

Reply:
Thanks for brining this up, it's very easy to miss, for some reason our theme has put it in a dropdown you can access by clicking the profile icon in the navbar. When I've used this before, it usually adds it in a more obvious spot in the navbar.

> Unrelated, Still keen to get away from Active admin eventually?
Yep, still very keen. We could maybe get the ball rolling with it and use this feature to stick it behind a feature flag.